packages/npms.sls

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+{% from "packages/map.jinja" import packages with context %}
+
+{% set req_states = packages.npms.required.states %}
+{% set req_pkgs = packages.npms.required.pkgs %}
+{% set wanted_npms = packages.npms.wanted %}
+{% set unwanted_npms = packages.npms.unwanted %}
+
+### REQ PKGS (without these, some of the WANTED PIPS will fail to install)
+npm_req_pkgs:
+  pkg.installed:
+    - pkgs: {{ req_pkgs | json }}
+
+### NPM PKGS to install using npm
+# (requires the npm deb/rpm installed, either by the system or listed in
+# the required packages
+{% for nn in wanted_npms %}
```

Comment:
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.npm.html#salt.states.npm.installed also has a `pkgs` option.  Can we avoid this loop and just use that instead?  Otherwise lots of extra, unnecessary noise if using this for numerous packages.